clients/command/client.go

```diff
@@ -56,13 +56,13 @@ func (c *CommandRestClient) init(params types.EndpointParams) {
 	}
 }
 
-// Get : issue GET command
-func (cc *CommandRestClient) Get(id string, cID string, ctx context.Context) (string, error) {
-	body, err := clients.GetRequest(cc.url+"/"+id+"/command/"+cID, ctx)
+// Get issues a GET command targeting the specified device, using the specified command
+func (cc *commandRestClient) Get(device string, command string, ctx context.Context) (string, error) {
```

Comment:
As we are using the IDs, maybe it's more clear to use `deviceID` and `commandID` instead of `device` and `command`.

Reply:
Done